test/optimize/test_optimize.py

```diff
@@ -436,3 +436,36 @@ def test_history_beats_optimizer():
         result_hist.optimize_result.list[0]['fval']
         < result_opt.optimize_result.list[0]['fval']
     )
+
+
+@pytest.mark.parametrize("local_optimizer", [None, optimize.FidesOptimizer()])
+@pytest.mark.flaky(reruns=3)
```

Comment:
bad start :)

Reply:
The alternative is checking only that the result is at all better than the starting points as done in the other tests or testing for some very high objective value...